server/utils.js

```diff
@@ -752,14 +796,49 @@ function getTimeBetweenDates(from, to) {
 function updateOccupancyLevels(occupancyLevel, date) {
   return {
     'occupancyLevels': {
+      'metadata': {
+        'timestamp': {
+          'type': 'date',
+          'value': date
+        }
+      },
       'type': 'PropertyValue',
-      'timestamp': date,
-      'name': 'occupancyLevels',
       'value': occupancyLevel
     }
   };
 }
 
+function asciiEncode(string) {
```

Comment:
is this really necessary?

encodeURIComponent can do the work and less code to maintain

a big -1 to include this kind of stuff